pkg/controllers/user.go

```diff
@@ -4,19 +4,27 @@ import (
 	"net/http"
 
 	"github.com/CESARBR/knot-babeltower/pkg/logging"
+
+	"github.com/CESARBR/knot-babeltower/pkg/interactors"
 )
 
 // UserController represents the controller for user
 type UserController struct {
-	logger logging.Logger
+	logger               logging.Logger
+	createUserInteractor *interactors.CreateUser
 }
 
 // NewUserController constructs the controller
-func NewUserController(logger logging.Logger) *UserController {
-	return &UserController{logger}
+func NewUserController(
+	logger logging.Logger,
+	createUserInteractor *interactors.CreateUser) *UserController {
+	return &UserController{logger, createUserInteractor}
 }
 
-// Create handles the server request
+// Create handles the server request and calls CreateUserInteractors
 func (uc *UserController) Create(w http.ResponseWriter, r *http.Request) {
 	// TODO: parse request
```

Comment:
Shouldn't this TODO be removed?

Reply:
Removed on this commit: 282b77d1434aa082b7d888ce3fbd03a28f4b09fe